src/LinuxTracing/LinuxTracingUtils.cpp

```diff
@@ -179,10 +168,12 @@ int GetTracepointId(const char* tracepoint_category, const char* tracepoint_name
   std::string filename = absl::StrFormat("/sys/kernel/debug/tracing/events/%s/%s/id",
                                          tracepoint_category, tracepoint_name);
 
-  std::optional<std::string> file_content = ReadFile(filename);
-  if (!file_content.has_value()) {
+  ErrorMessageOr<std::string> file_content = orbit_base::ReadFileToString(filename);
+  if (!file_content) {
+    ERROR("Reading tracepoint id: %s", file_content.error().message());
```

Comment:
What about printing the tracepoint too, e.g.:
`ERROR("Reading tracepoint id of  %s:%s: %s", tracepoint_category, tracepoint_name, file_content.error().message());`

Reply:
done.